docs/07-sales-and-marketing/services.md

```diff
@@ -0,0 +1,57 @@
+# Services (WIP)
+
+## Application Migration
+* Drupal Application Migration
+
+## Cloud-Based Infrastructure
+* Systems Administration (SA)
+* Hosting Services
+
+## Design
+* User Research
+* Content Strategy
+* Information Architecture
+* Interaction Design and Front End Development
+* Visual Design
+
+## Development
```

Comment:
I think they should be separated, and yes that gives us a place to list specific front end competencies @nerdstein